hotfixes/4.0.1.cfg

```diff
@@ -25,6 +26,7 @@ Products.PloneHotfix20131210 = 1.0
 Products.PloneHotfix20150910 = 1.1
 Products.PloneHotfix20151208 = 1.0
 Products.PloneHotfix20160419 = 1.0
+Products.PloneHotfix20160830 = 1.1
```

Comment:
@lukasgraf I'll update the hotfix version (for Plone versions). Do you want to re-deploy afterwards?